_slides/19-scarowar.md

```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+# The Octocat
```

Comment:
liked the chore() format :D
can you please make it your name?

Reply:
Yes, the pull request has been changed as requested ^^